closed/src/java.base/share/classes/sun/security/ec/NativeECKeyPairGenerator.java

```diff
@@ -192,13 +211,26 @@ public void initialize(AlgorithmParameterSpec params, SecureRandom random)
         }
 
         // Not all known curves are supported by the native implementation
-        ECKeyPairGenerator.ensureCurveIsSupported(ecSpec);
+        if (!(NativeECUtil.isBrainpoolP512r1(ecSpec))) {
```

Comment:
Please remove the redundant parentheses.

Reply:
Redundant parentheses removed.